client/js/components/procedure/StatementSegmentsList/StatementSegmentsList.vue

```diff
@@ -69,7 +69,7 @@
           <li>
             <dp-button
               class="u-ph-0_25"
-              :href="Routing.generate('dplan_segments_export', { procedureId: procedureId, statementId: statementId })"
+              @click="showHintAndDoExport('dplan_segments_export')"
```

Comment:
Why do you pass the route name here? it is not used within the fn body.

nitpicky question: while `showHintAndDoExport` pretty much sums up what the fn does, i'd rather chose the more handy `exportStatements` as fn name

Reply:
I just messed it up. Changed in d588a994814e4dae4257378d82e38773058b4566

Reply:
> i'd rather chose the more handy exportStatements as fn name
I think I would stick to the name hence its the same as the other place. And If we want to move the Code out to some external fn its easier to find all occurrences